addons/calendar/models/calendar_event.py

```diff
@@ -747,6 +765,28 @@ def find_partner_customer(self):
     # TOOLS
     # ------------------------------------------------------------
 
+    def _find_attendee(self):
```

Comment:
I really don't understand this. Why is my attendee status based on contact computing ?